airflow/dags/gtfs_schedule_history2/merge_updates.py

```diff
@@ -83,9 +84,11 @@ def merge_updates(table_name, execution_date, **kwargs):
 
 
 def main(**kwargs):
+    logger = structlog.get_logger()
     table_names = create_tables()
 
     for table_name in table_names:
+        logger.info("Processing {}".format(table_name))
```

Comment:
with structlog you can pass kwargs to provide key-value pairs, for example `logger.info('processing table', table_name=table_name)`

Reply:
addressed in b2c295a20e63c35761223a27b7a8c0684d060cd2